.changeset/eight-pianos-divide.md

```diff
@@ -0,0 +1,28 @@
+---
+'@xstate/graph': major
+---
+
+pr: #4896
+commit: 7c6e2ea
```

Comment:
just remember to add this to other changeset files here - i think at least most of them should have this but u'll be able to tell best